README.md

```diff
@@ -306,6 +307,18 @@ Check out the built-in code steps [here](src/main/scala/com/yotpo/metorikku/code
 
 *NOTE: If you added some dependencies to your custom JAR build.sbt you have to either use [sbt-assembly](https://github.com/sbt/sbt-assembly) to add them to the JAR or you can use the ```--packages``` when running the spark-submit command* 
 
+#### Watermark
+Metorikku supports Watermark method which helps a stream processing engine to deal with latenes.
+You can use watermarking by adding a new udf step in your metric:
+```yaml
+- dataFrameName: dataframe
+  classpath: com.yotpo.metorikku.code.steps.Watermark
+  params:
+    table: my_table
+    eventTime: event
```

Comment:
Comment: The column representing the event time (needs to be a TIMESTAMP or DATE column)